modules/howtos/examples/TransactionsExample.java

```diff
@@ -64,7 +70,7 @@
     public static void main(String... args) {
         // tag::init[]
         // Initialize the Couchbase cluster
-        Cluster cluster = Cluster.connect("localhost", "Administrator", "password");
+        Cluster cluster = Cluster.connect("localhost", "username", "password");
```

Comment:
thanks for fixing this anti-pattern.  :)

Reply:
I thought you'd like this :)